ci/travis-after-success.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xe
 
+# Submit code coverage report
+curl -s https://codecov.io/bash | bash
```

Comment:
OK, the script is opensource and stored in GitHub. I would prefer to use a GitHub URL using a particular hash to download it or even just shipping the script as part of the `sociomantictsunami/base` cachalot image, so it is available to all projects (although for now downloading might be better until we decide to go this route or not).

But I was just looking at the scripts and they are really awful. Is not making me trust codecov much really, in the python upload script they parse the yaml manually through regex... WTF. Also they do all sort of guessing for all the types or CIs out there and searching for all type of coverage reports out there, when all needed should be just a simple REST call to a website to send a report, like any other sane service out there...

I really don't understand why they do this and I don't like it at all...